spec/helpers/hearing_paginator_spec.rb

```diff
@@ -31,10 +31,10 @@
 
   let(:fake_hearings) do
     [
-      CdApi::HearingSummary.new(id: 'hearing-uuid-3').tap { _1.day = hearing3_days.first.to_datetime },
-      CdApi::HearingSummary.new(id: 'hearing-uuid-1').tap { _1.day = hearing1_days.first.to_datetime },
-      CdApi::HearingSummary.new(id: 'hearing-uuid-1').tap { _1.day = hearing1_days.last.to_datetime },
-      CdApi::HearingSummary.new(id: 'hearing-uuid-2').tap { _1.day = hearing2_days.first.to_datetime }
+      build(:hearing_summary, id: 'hearing-uuid-3').tap { _1.day = hearing3_days.first.to_datetime },
```

Comment:
I am not totally sure why we do need to change the day after we build the `fake_hearings`,
do we still need it?

Reply:
Good spot! We can set the day as an attribute in the `build` call.